examples/custom-hooks/custom-hooks.js

```diff
@@ -0,0 +1,40 @@
+// This example showcases the custom hooks API
+// buildHuffmanTable() gets called quite often and only logs to console from time to time,
+//  skipping the call to the original function
+// copyToImageData() gets called eventually and causes an error
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const hooking = require("@jazzer.js/hooking");
+
+module.exports.buildHuffmanTableHook = hooking.hookManager.registerHook(
```

Comment:
I pushed a commit that shows how a changed export in `hooking` would simplify the hooks. Feel free to remove this later again.